maro/rl/policy/policy.py

```diff
@@ -60,6 +62,19 @@ def get_rollout_info(self):
     def get_batch_loss(self, batch: dict, explicit_grad: bool = False):
         pass
 
+    def data_parallel(self, *args, **kwargs):
+        self.remote = True
+        self._proxy = Proxy(*args, **kwargs)
+
+    def data_parallel_with_existing_proxy(self, proxy):
```

Comment:
Why do we need this if we already have data_parallel?

Reply:
For the arguments in the interface.
If we already have a `proxy`, the other arguments in `data_parallel` is useless.

Reply:
But it deserves to think how to unify the interface.

Reply:
It would be best if you could figure out a way to unify them.

Reply:
How about:
```
def data_parallel(self, **kargs):
    if 'proxy' in kargs:
        self._proxy = ...
    else:
        ....
```

Also, why do we need both args and kargs at the sametime?

Reply:
Brilliant! Args can be hidden in kwargs and aquired manually.